java/src/jmri/jmrit/operations/router/Router.java

```diff
@@ -44,6 +50,7 @@ public class Router extends TrainCommon implements InstanceManagerAutoDefault {
     private final List<Train> _lastLocationTrains = new ArrayList<>();
 
     protected static final String STATUS_NOT_THIS_TRAIN = Bundle.getMessage("RouterTrain");
+    public static final String STATUS_NOT_THIS_TRAIN_PREFIX = Router.STATUS_NOT_THIS_TRAIN.substring(0, 5);
```

Comment:
This works fine with English but it seems prone to failure with other languages. For example, in Swedish `RouterTrain` would be:
`RouterTrain             = Tåg ({0})`

where `Train` in Swedish is `Tåg` which is only three characters. But maybe it's no big deal.

But a better solution might be:
`Router.STATUS_NOT_THIS_TRAIN.substring(0, Router.STATUS_NOT_THIS_TRAIN.indexOf('(') - 1);`

Reply:
Thanks and I agree that's a better solution.  Having the prefix include up to the left parenthesis is also okay.